src/components/dialogs/network-modifications/line/characteristics-pane/line-connectivity-pane.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { Grid } from '@mui/material';
+import { gridItem, GridSection } from '../../../dialogUtils';
+import { ConnectivityForm } from '../../../connectivity/connectivity-form';
+import { CONNECTIVITY_1, CONNECTIVITY_2 } from 'components/utils/field-constants';
+import React, { FunctionComponent } from 'react';
+import useVoltageLevelsListInfos from '../../../../../hooks/use-voltage-levels-list-infos';
+import { CurrentTreeNode } from '../../../../../redux/reducer';
+import { UUID } from 'crypto';
+
+interface LineConnectivityPaneProps {
```

Comment:
You can rename this component to LineConnectivityForm and move to dialogs/connectivity. This is a shared component by 2TW and Line so it is better to move this component in a separated folder